src/gui/rss/articlelistwidget.cpp

```diff
@@ -123,11 +123,15 @@ QListWidgetItem *ArticleListWidget::createItem(RSS::Article *article) const
     item->setData(Qt::DisplayRole, article->title());
     item->setData(Qt::UserRole, reinterpret_cast<quintptr>(article));
     if (article->isRead()) {
-        item->setData(Qt::ForegroundRole, QPalette().color(QPalette::Inactive, QPalette::WindowText));
+        const QColor defaultColor {palette().color(QPalette::Inactive, QPalette::WindowText)};
+        const QBrush foregroundRole {UIThemeManager::instance()->getColor("Rss.Read", defaultColor)};
```

Comment:
RSS.ReadArticle

Reply:
"foregroundRole" doesn't really mean any "role" but a color/brush associated with "foreground role". So I would rename it to "foregroundColor" or "foregroundBrush" or even to "brush" since its "foreground role" is clearly seen from the context and there is no other brushes within the same context.